phpstan-baseline.neon

```diff
@@ -0,0 +1,672 @@
+parameters:
```

Comment:
Lässt sich der Pfad ändern? Dann würde ich die in `.tools` oder ggf. in `.tools/phpstan` legen (wir haben schon einen Ordner `.tools/psam`).

Reply:
Ja, der pfad wird in der phpstan.neon.dist Festgelegt